go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/container-storage-interface/spec v1.10.0
 	github.com/go-kit/log v0.2.1
 	github.com/golang/protobuf v1.5.4
-	github.com/grpc-ecosystem/go-grpc-middleware v1.4.0
+	github.com/grpc-ecosystem/go-grpc-middleware/v2 v2.1.0
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
```

Comment:
https://github.com/grpc-ecosystem/go-grpc-prometheus

This was deprecated, and the functionality is now available from `github.com/grpc-ecosystem/go-grpc-middleware/v2`.

Reply:
It is now replaced with the new module: `github.com/grpc-ecosystem/go-grpc-middleware/providers/prometheus v1.0.1`